boards/arm/nrf52840_m2/nrf52840_m2_defconfig

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) 2022 Stavros Avramidis <stavros9899@gmail.com>
+# SPDX-License-Identifier: Apache-2.0
+
+CONFIG_SOC_SERIES_NRF52X=y
+CONFIG_SOC_NRF52840_QIAA=y
+CONFIG_BOARD_NRF52840_M2=y
+
+
+# Enable MPU
+CONFIG_ARM_MPU=y
+
+# Enable hardware stack protection
+CONFIG_HW_STACK_PROTECTION=y
+
+# Enable RTT
+CONFIG_USE_SEGGER_RTT=y
```

Comment:
Why or how would someone use segger RTT with a DAPLINK?

Reply:
`pyocd rtt` for example